les/peer.go

```diff
@@ -732,7 +730,6 @@ func (p *serverPeer) answeredRequest(id uint64) {
 	}
 	e, ok := p.sentReqs[id]
 	delete(p.sentReqs, id)
-	vt := p.valueTracker
```

Comment:
This function can be modified a bit, which only returns the serving time. All the value calculation work can be thrown to `vflux/client/serverpool.go`